src/CsharpMacros/Macros/RangeMacro.cs

```diff
@@ -14,13 +14,24 @@ class RangeMacro:ICsharpMacro
             {
                 var from = int.Parse(matchedRange.Groups["from"].Value);
                 var to = int.Parse(matchedRange.Groups["to"].Value);
-                for (int i = from; i <= to; i++)
+                var step = 1;
+                if (matchedRange.Groups["step"].Success)
+                {
+                    step = int.Parse(matchedRange.Groups["step"].Value);
+                    if (step < 1 || from > to)
+                    {
+                        // Not sure if an exception should be thrown here.
```

Comment:
Can we throw exceptions here?

Reply:
Unfortunately, the decent error handling mechanism has not been implemented yet so I would rather avoid throwing an exception.  If the step has an invalid value you should fallback to default.